### PR TITLE
Document some path functions

### PR DESCRIPTION
**Pull request by A2va**
* Before adding new features and new modules, please go to issues to submit the relevant feature description first.
* Write good commit messages and use the same coding conventions as the rest of the project.
* Please commit code to dev branch and we will merge into master branch in feature
* Ensure your edited codes with four spaces instead of TAB.

------

* 增加新特性和新模块之前，请先到issues提交相关特性说明，经过讨论评估确认后，再进行相应的代码提交，避免做无用工作。
* 编写友好可读的提交信息，并使用与工程代码相同的代码规范，代码请用4个空格字符代替tab缩进。
* 请提交代码到dev分支，如果通过，我们会在特定时间合并到master分支上。
* 为了规范化提交日志的格式，commit消息，不要用中文，请用英文描述。

### CODE REVIEW

File: manual/builtin_modules.md [hunk 1]
```diff
@@ -984,7 +984,7 @@ os.setenv("HOME", "/tmp/")
 Consistent with the result of [$(tmpdir)](#var-tmpdir), it is just a direct return to return a variable that can be maintained with subsequent strings.
 
 ```lua
-print(path.join(os.tmpdir(), "file.txt"))
+print((os.tmpdir(), "file.txt"))
```
Comment (waruqi) [inline manual/builtin_modules.md:987]: why remove join?

Comment (A2va) [reply manual/builtin_modules.md]: It was a mistake

**Comment by A2va** [comment]
For the supported versions of these functions, I have done so in relation to similar functions. For example the env functions were added in version 2.2.7 so the `path.joinenv` function also comes from this version. Because of this I'm not sure if all of them are correct.

**Comment by waruqi** [comment]
thanks

**Pull request merged**